royalties-registry/migrations/5_royalties_provider_art_blocks.js

```diff
@@ -0,0 +1,78 @@
+const RoyaltiesRegistry = artifacts.require('RoyaltiesRegistry');
+const RoyaltiesProviderArtBlocks = artifacts.require("RoyaltiesProviderArtBlocks");
+
+const ZERO = "0x0000000000000000000000000000000000000000";
+
+const rinkeby = {
+    tokens: [],
+    artblocksAddress: ""
+}
+const mainnet = {
+    tokens: [
+        "0x059edd72cd353df5106d2b9cc5ab83a52287ac3a",
+        "0xa7d8d9ef8d8ce8992df33d8b8cf4aebabd5bd270"
+    ],
+    artblocksAddress: "0x6C093Fe8bc59e1e0cAe2Ec10F0B717D3D182056B"
+}
+const ropsten = {
```

Comment:
may be there are artblocks on rinkeby or ropsten?

Reply:
added rinkeby address